### PR TITLE
Add news section

### PR DESCRIPTION
**Pull request by daisylb**

### CODE REVIEW

**Review by devjack — APPROVED** [review]
This is great @excitedleigh.

**Comment by devjack** [comment]
@excitedleigh new commit adds the /news index for posts. We can add navigation later as these posts are linked to from /about already.

File: templates/news_index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+{% import '_macros.html' as m %}
+{% block main %}
+<h1>News &amp; Announcements</h1>
+<ul style="list-style-type: none;">
+    {% for post in news %}
+    <li style="margin-bottom:1em;">
+        <p>
+            <a href="{% url "news_post", post %}">{{ post.title }}</a>
+            <br />
+            {%- if loop.first -%}<b>Latest News</b> &mdash; {% endif %}<small>Published {{ post.published.strftime("%d %B, %Y") }}</small>
```
Comment (daisylb) [inline templates/news_index.html:11]: I'm not sure this is necessary; it's already clear that it's the latest by virtue of it being at the top of the list, right?

**Comment by daisylb** [comment]
Ironically, I can't approve this because I wrote the original PR. But there's nothing stopping me from merging anyway, so _yeet_

**Pull request merged**